src/css/components/_progress.scss

```diff
@@ -173,3 +169,7 @@
   color: #fff;
   @include background-color-with-alpha(#000, 0.8);
 }
+
+.vjs-live .vjs-load-progress {
+  display: none;
```

Comment:
I am not sure if we want to hide this or fix it, basically this can make it so that the progress bar goes past the player.